.release/release-metadata.hcl

```
@@ -0,0 +1,3 @@
+url_license = "https://github.com/hashicorp/vault-lambda-extension/blob/main/LICENSE"
+url_project_website = "https://learn.hashicorp.com/tutorials/vault/aws-lambda?in=vault/app-integration"
```

```suggestion
url_project_website = "https://github.com/hashicorp/vault-lambda-extension#readme"
```

Until we get a section in the platform part of the docs website, the readme is the official documentation source